var/spack/repos/builtin/packages/py-cupy/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCupy(PythonPackage):
+    """CuPy is an open-source array library accelerated with
+    NVIDIA CUDA. CuPy provides GPU accelerated computing with
+    Python. CuPy uses CUDA-related libraries including cuBLAS,
+    cuDNN, cuRand, cuSolver, cuSPARSE, cuFFT and NCCL to make
+    full use of the GPU architecture."""
+
+    homepage = "https://cupy.dev/"
+    url = "https://pypi.io/packages/source/c/cupy/cupy-8.0.0.tar.gz"
+
+    version('8.0.0', sha256='d1dcba5070dfa754445d010cdc952ff6b646d5f9bdcd7a63e8246e2472c3ddb8')
+
+    depends_on('python@3.6:', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.5:', type=('build', 'run'))
```

Reply:
They don't advise that and this requires setting an environment var to override the safety. Is this necessary?

https://github.com/cupy/cupy/blob/36aa7523edd203f0692845495e9bca0f7b5702fa/setup.py#L24-L33

Reply:
Oh right, you're correct.